stable/postgresql/templates/initialization-configmap.yaml

```diff
@@ -7,8 +7,13 @@ metadata:
     chart: {{ template "postgresql.chart" . }}
     release: {{ .Release.Name | quote }}
     heritage: {{ .Release.Service | quote }}
+binaryData:
```

Comment:
binaryData configMaps are supported since Kubernetes version 1.10.0, can you increase the Kubernetes version prerequisite in the README?

https://github.com/helm/charts/blob/master/stable/postgresql/README.md#prerequisites

Reply:
👍